source/content/supported-drupal.md

```diff
@@ -16,12 +16,18 @@ The following table indicates availability of the specified Drupal version, as w
 
 | Drupal Version | Available | Recommended | Supported |
 | ----------- | :---------: | :---------: | :---------: |
+| 11          | ⚠️ <Popover title="Drupal 11 Availability" content="See the <a href='#drupal-11-on-pantheon'>following section</a> for how to run Drupal 11 on Pantheon today." />        | <span style="color:green">✔</span>           | <span style="color:green">✔</span>
 | 10          | <span style="color:green">✔</span>         | <span style="color:green">✔</span>           | <span style="color:green">✔</span>          |
 | 9           | <span style="color:green">✔</span>          | <span style="color:green">✔</span>           | <span style="color:green">✔</span>         |
 | 8           | <span style="color:green">✔</span>          | ❌           | <span style="color:green">✔</span>         |
 | 7           | <span style="color:green">✔</span>         | ❌           | <span style="color:green">✔</span>          |
 | 6           | ❌          | ❌           | ❌          |
 
+## Drupal 11 on Pantheon
+Drupal 11 is not yet an option when creating a new site and there is no current ETA for when it will be added. However, Drupal 11 has been tested and is reported to work on Pantheon today.
```

Comment:
I don't like calling out that we don't have an ETA. It feels better to just say "it's coming", which is still true. 
Updated the language from "reported to work" to something that sounds more confident.

```suggestion
Drupal 11 is not yet an option when creating a new site. It will be added to the Pantheon dashboard in the coming weeks. However, Drupal 11 has been tested and works on Pantheon environments.
```